include/igl/opengl/glfw/imgui/ImGuiMenu.cpp

```diff
@@ -113,7 +113,9 @@ IGL_INLINE bool ImGuiMenu::mouse_down(int button, int modifier)
 
 IGL_INLINE bool ImGuiMenu::mouse_up(int button, int modifier)
 {
-  return ImGui::GetIO().WantCaptureMouse;
+  //return ImGui::GetIO().WantCaptureMouse;
+  // !! Should not steal mouse up
```

Comment:
Why this change?

Reply:
This is just general GUI good manners. Don't steal mouse up. Everyone should be allowed to know if mouse up happened.

Reply:
I'm not sure about that. Why is that different from a mouse down event? This flag means that mouse event happens on a imgui widget, so it should not be dispatched/may be hidden from the rest of the application. But maybe you have a specific example of interaction where this was problematic?